modin/engines/ray/utils.py

```diff
@@ -119,6 +122,8 @@ def initialize_ray(cluster=None, redis_address=None, redis_password=None):
                 memory=object_store_memory,
                 lru_evict=True,
             )
+        else:
+            raise ValueError("cluster variable should be 'True' or None")
```

Comment:
now I am a bit lost on the function API... I second the request on docstring for this function.